src/asset/asset.js

```diff
@@ -61,7 +61,7 @@ Object.assign(pc, function () {
      * var asset = new pc.Asset("a texture", "texture", {
      *     url: "http://example.com/my/assets/here/texture.png"
      * });
-     * @param {object} [options] - a JSON object containing load-time options specific to the asset.
+     * @param {(pc.TextureHandlerOptions|pc.ContainerHandlerOptions)} [options] - The asset handler options.
```

Comment:
Why not just keep this as a generic `object`?

Reply:
done